tests/cypress/tests-e2e/featureSelection.cy.js

```diff
@@ -445,5 +445,84 @@ describe('Testing the feature selection', () => {
             cy.get('@identifySingleFeature.all').should('have.length', 1)
             cy.get('@emptyIdentify.all').should('have.length', 1)
         })
+
+        it('can print feature information', () => {
+            // Import KML file
+            const fileName = 'external-kml-file.kml'
+            const localKmlFile = `tests/cypress/fixtures/import-tool/${fileName}`
+            cy.goToMapView(
+                {
+                    layers: 'test.wms.layer',
+                },
+                true
+            )
+            cy.wait(['@routeChange', '@layers', '@topics', '@topic-ech'])
+
+            cy.openMenuIfMobile()
+            cy.get('[data-cy="menu-tray-tool-section"]:visible').click()
+            cy.get('[data-cy="menu-advanced-tools-import-file"]:visible').click()
+            cy.get('[data-cy="import-file-local-btn"]:visible').click()
+            cy.readFile(localKmlFile, null).as('printKmlFixture') // .readFile instead of .fixture because .fixture caches the file and makes it impossible to re-read it in another test
+            cy.get('[data-cy="file-input"]').selectFile('@printKmlFixture', {
+                force: true,
+            })
+            cy.get('[data-cy="import-file-load-button"]:visible').click()
+            cy.get('[data-cy="file-input-text"]').should('contain.value', fileName)
+
+            cy.wait(['@icon-sets', '@icon-set-babs', '@icon-set-default'])
+
+            cy.get('[data-cy="import-file-close-button"]:visible').realClick()
```

Comment:
is it really needed to use a `realClick` here? if we can avoid binding lots of our code to this lib, it would be nice (for when we might need to update/remove it)